FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -658,12 +677,20 @@ void Maslow_::update(){
             home();
 
         }
+        else { //In any other state, keep motors off
+        
+        
+        digitalWrite(coolingFanPin, HIGH);
```

Comment:
Couldn't get this to run at the right times, so for now it's just on all the time